src/store/flow/flow.ts

```diff
@@ -171,7 +171,15 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
   async flow_persist({ state, getters, commit }, { persistRoute, flowContainer }): Promise<IContext> {
     const restVerb = flowContainer.created ? 'put' : 'post'
     const oldCreatedState = flowContainer.created
-    return axios[restVerb](persistRoute, omit(flowContainer, ['created']))
+
+    const flowContainerCleanedResources = cloneDeep(flowContainer)
+    flowContainerCleanedResources.resources = []
+    flowContainer.resources.forEach((resource, i) => {
+      if(JSON.stringify(flowContainer.flows).includes(resource.uuid)) {
```

Comment:
Can we pull the JSONification out of the loop to mitigate slow-by-slow looping? 😛